project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.17
```

Comment:
Why not 1.x?

Reply:
At that time sbt 1.0 had file watching issues on macs, I think.
But they should be fixed with the latest sbt 1.0.6.